mmvb_backend/case_synthesizer/api/urls.py

```diff
@@ -0,0 +1,11 @@
+from rest_framework.routers import DefaultRouter
+
+from case_synthesizer.api.views import CaseSynthesizerViewSet
+from case_synthesizer.apps import CaseSynthesizerConfig
+
+app_name = CaseSynthesizerConfig.name
+
+router = DefaultRouter(trailing_slash=False)
```

Comment:
nice, does this solve the trailing slash problem? Does that mean everything can now POST / GET endpoints can be called with and without final slash?

Reply:
Actually we moved into the direction of having all the endpoints without trailing slashes. So even for paths that represent actions (`/api/v1/cases/synthesize` for example) there will be no trailing slash.